datavalid.yml

```diff
@@ -60,6 +60,8 @@ schemas:
           - female
           - male
       - name: agency
+      - name: years_of_service
+      - name: rank_desc 
```

Comment:
I think you should put `years_of_service` in the `event` table instead. `rank_desc` might change for an officer so it should not be added to `personnel`.